src/invoice_utils/web.py

```diff
@@ -105,6 +115,13 @@ def generate_invoice(request: InvoiceRequest):
     return context
 
 
+def _get_default_rules(root_dir):
+    basic_rules = root_dir / "basic.json"
+    # TODO: Change this in phase 2 by using repository
```

Comment:
Please remove this comment. Add an issue in the tracker or to a personal reminders software instead of this comment here. Inline comments should be useful information about the code that _already exists_ in the repository. They usually explain complex algorithms and/or design choices.